core/src/main/java/cn/edu/tsinghua/iot/benchmark/conf/ConfigDescriptor.java

```diff
@@ -721,6 +721,8 @@ private boolean checkDatabaseVerification(DBConfig dbConfig) {
     } else if (dbConfig.getDB_SWITCH() == DBSwitch.DB_TIMESCALE) {
       // support timescaledb
       result = true;
+    } else if (dbConfig.getDB_SWITCH() == DBSwitch.DB_CNOS) {
+      result = true;
```

Comment:
@SpriCoder cc whether it supports Verification?

Reply:
Yes